dev/config.js

```diff
@@ -63,6 +63,10 @@
       controller: 'profileCtrl',
       controllerAs: 'vm',
       templateUrl: 'profile/tpl.html'
+    }).when('/template/:id/:templateId', {
```

Comment:
I need one url for two differents page, if :id===null i show one page, if templateId===null i show another